matrix/rules.py

```diff
@@ -345,7 +368,7 @@ def exception_handler(self, context, loop=None, exc_ctx=None):
 
     async def __call__(self):
         btask = self.loop.create_task(self.bus.notify(False))
-        context = self.load_suite(self.config_file)
+        context = await self.load_suite()
```

Comment:
I appreciate the run_in_executor hurdle you jumped here (and we can keep it) but the setup code here doesn't strictly need to be async as nothing else can run when this is called.